sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner.py

```diff
@@ -358,8 +358,11 @@ def run_stages(self,
         stage_context.components.environments, self._provision_info)
     pipeline_metrics = MetricsContainer('')
     pipeline_metrics.get_counter(
-        MetricName(str(type(self)),
-                   self.NUM_FUSED_STAGES_COUNTER)).update(len(stages))
+        MetricName(
+            str(type(self)),
+            self.NUM_FUSED_STAGES_COUNTER,
+            urn='internal:' + self.NUM_FUSED_STAGES_COUNTER)).update(
+                len(stages))
```

Comment:
I guess `internal:` is a concept already defined somewhere?

Reply:
Yeah, there's some inconsistencies here with "user" counters requiring a name and namespace and "non-user" counters using the first component of the urn as a namespace and the rest as the name.